src/components/CategoryItem/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import { TouchableOpacity, Image, Text, View } from 'react-native';
+import styles from './CatItemStyle';
+
+
+
+export default class CategoryItem extends Component {
+
+
+  goToTaskScreen(){
+    this.props.goToTask();
+}
+
+
+  render(){
+    const {id,title,img} = this.props;
+    //icon = require
+    return (
+        <TouchableOpacity style={styles.rectangle} onPress={ this.goToTaskScreen.bind(this) }>
```

Comment:
Vas a necesitar pasar la categoria aqui
`<TouchableOpacity style={styles.rectangle} onPress={ () => this.goToTaskScreen(img) }>`
O si ya renombraste img por category (valor que te sirve para la imagen tambien)
`<TouchableOpacity style={styles.rectangle} onPress={ () => this.goToTaskScreen(category) }>`